delphi-checks/src/main/resources/org/sonar/l10n/delphi/rules/community-delphi/FullyQualifiedImport.json

```
@@ -0,0 +1,18 @@
+{
+  "title": "Imports should be fully qualified",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
```

No need to escape this forward slash. 
```suggestion
    "func": "Constant/Issue",
```

---

Done!